packages/venia-concept/src/util/__tests__/makeUrl.spec.js

```diff
@@ -30,18 +30,12 @@ test('prepends media path for product images', () => {
     expect(makeUrl(relativePath, { type: 'image-product' })).toBe(
         `${productBase}${relativePath}?${defaultParams}`
     );
-    expect(makeUrl(absoluteUrls[2], { type: 'image-product' })).toBe(
```

Comment:
Test changes all related to removed functionality. See https://github.com/magento-research/pwa-studio/pull/1267/files#r287155267